test/langtools/jdk/jshell/UserInputTest.java

```diff
@@ -47,7 +47,7 @@ public void testReadInput() {
 
     public void testScanner() {
         assertEval("import java.util.Scanner;");
-        assertEval("Scanner s = new Scanner(System.in);");
+        assertEval("Scanner s = new Scanner(System.in, System.getProperty(\"stdin.encoding\"));");
```

Comment:
JShell's `System.in` is not a real `System.in` (set by `JShell.builder().in()`), so I think this one too is not necessary